### PR TITLE
13585 user dropdown styling

### PR DESCRIPTION
**Pull request by rianfowler**
## Description

- on the drop down / button: use the width of whichever is longer - the name or the dropdown items
- remove all new buttons
- remove the icon to external links for My Health

## Testing done
verified locally 

## Screenshots
screen shot 2018-09-26 at 16 02 00
screen shot 2018-09-26 at 16 01 53

## Definition of done
~- [ ] Events are logged appropriately~
~- [ ] Documentation has been updated, if applicable~
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/sass/merger.scss [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 /* OVERRIDE */
 .merger {
+  .va-dropdown-trigger {
+    min-width: 102px;
```
Comment (rianfowler) [inline src/platform/site-wide/sass/merger.scss]: not a fan of magic numbers but this was what I came up with without altering the markup. The drop down is in a separate container from the button so it's tricky to dynamically get this width without js.

**Review by cehsu — APPROVED** [review]
LGTM. Thanks for the context.

**Comment by rianfowler** [comment]
@emilywaggoner I didn't alter any of the font stuff for this ticket since this dropdown using mostly global font settings. 

45988037-eb651180-c031-11e8-9ae9-2f372125fecf

**Review by emilywaggoner — APPROVED** [review]
Thanks @rianfowler! Looks great.

**Review by erikphansen — APPROVED** [review]

**Pull request merged**